src/components/GlobalStyle.js

```diff
@@ -4,6 +4,7 @@ const GlobalStyle = createGlobalStyle`
   body {
     background-color: ${props => props.theme.backgroundColor};
     overflow-y: scroll;
+    -webkit-font-smoothing: antialiased;
```

Comment:
What is this doing and is it necessary? I've heard some arguments why this shouldn't be used, but I don't have up to date info on what the current best practices are.

This might make fonts look better on MacOS etc., but is it harming other browsers?

Reply:
@kpuputti interesting! First time I hear about this and we have been using this everywhere, FTW, Console, marketing-site etc. This makes the fonts look crispier and thus increases the readability because of the crispier edge of the pixels. 

I'll suggest that we put it in now, and if it causing issues in Windows or Linux, then we can always target only MacOs individually with this.

Reply:
I remember researching it a bit earlier as well, and maybe even discussing with @Gnito . Not that I'm not only referring to all browsers on MacOS, different browsers might behave differently also on MacOS.

The usual accessibility argument is that browsers should know by default what's the best way to render stuff. But if this has clear visible benefits, let's just keep it for now.

Reply:
@kpuputti visually I think it has a clear benefit. In my opinion, only Safari can render the fonts correctly and all other browsers aren't so great out of the box.